packages/html/package.json

```diff
@@ -7,11 +7,8 @@
     "dev": "tsc --watch --preserveWatchOutput",
     "test": "mocha --no-config --require ts-node/register src/index.test.ts"
   },
-  "dependencies": {
-    "ejs": "^3.1.6"
-  },
   "devDependencies": {
-    "@types/ejs": "^3.1.0",
+    "@prairielearn/tsconfig": "*",
```

Comment:
Is this `tsconfig` package also released to `npm`?

Reply:
Nope, it's marked as private: https://github.com/PrairieLearn/PrairieLearn/pull/5654/files#diff-d75b7a6d7a2f99f72c90ad29d2d4953ec3ad2f4640af5285b1886c1a85bdf674R4

Reply:
Ah! Very good.